bin/common.py

```diff
@@ -137,21 +138,36 @@ def get_vault_admin_client(consul, customer_uuid):
     return VaultCA(url, token, ca_name, ca_common_name)
 
 
-def create_host_signing_role(vault, consul, customer_id):
+def create_host_signing_role(vault, consul, customer_id) -> str:
     rolename = 'hosts-%s' % customer_id
-    vault.create_signing_role(rolename)
-    consul.kv_put('customers/%s/vouch/ca_signing_role' % customer_id,
-                  rolename)
-    return rolename
+    customer_key: str = f'customers/{customer_id}/vouch/ca_signing_role'
+    try:
+        val = consul.kv_get(customer_key)
+        LOG.debug('kv_get for %s returned: %s', customer_key, val)
+        return rolename
+    except HTTPError as err:
+        LOG.error('cannot do kv_get on %s', customer_key, exc_info=err)
```

Comment:
do you need to reraise this exception for the non 404 case?

Reply:
done